jasmine-standalone-3.4.0/src/game.js

```diff
@@ -0,0 +1,24 @@
+
+var BowlingGame = function (){
+  this.rolls = [];
+}; //define a bowling game
+
+BowlingGame.prototype.roll = function(pins) {
+  this.rolls.push(pins);
+};
+
+BowlingGame.prototype.score = function () {
+  var result = 0;
+  var rollIndex = 0;
+
+  for (var frameIndex = 0; frameIndex < 10; frameIndex++) {
```

Comment:
I'm intrigued by frameIndex here. How does frameIndex know that there are 2 rolls per frame? As it stands this loop reads (I may be wrong) but that it only looks at the first 10 rolls. In your tests because the majority of rolls are padded with 0s this might be hiding something under the surface. I'd suggest writing out a full game test to explore this.

Reply:
Ah - is the 10 frames coming in because rollIndex is incremented by 2 each time?